staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -94,3 +100,13 @@ func (a *AdmissionOptions) Validate() []error {
 	errs := []error{}
 	return errs
 }
+
+// SetDefaultPlugins sets the PluginNames field if it was not provided.
+// The field is populated from the hard coded list of default plugin names.
+// Note: This method MUST be called after AddFlags
+func (a *AdmissionOptions) SetDefaultPlugins() {
+	if len(a.PluginNames) > 0 {
+		return
+	}
+	a.PluginNames = defaultPluginNames
```

Comment:
for instance we could remove some plugins thus effectively implement "turning off" functionality.